prge_01_tk/src/App.js

```diff
@@ -0,0 +1,21 @@
+import "./App.css";
+
+const lista_uzytkownikow = [
+  { imie: "Marek" },
+  { imie: "Wika" },
+  { imie: "Kinia" },
+];
+
+console.log(lista_uzytkownikow);
```

Comment:
Why in the frozen acid hell would you add a console log with user list? Tis a bloody leak for crying out loud